src/utils.ts

```diff
@@ -195,6 +195,16 @@ const isAuthorizationCodeGrantConfig = (config: OAuthConfig): config is Authoriz
 const isRefreshGrantConfig = (config: OAuthConfig): config is RefreshGrantConfig =>
   config.grantType === OAuthGrantType.REFRESH_TOKEN_GRANT;
 
+const transformError = (errorToTransform: any) => {
+  let transformedError;
```

Comment:
I would rather go for returning the values from the try-catch-block

Reply:
I thought for the user it is not of interest why the error could not be stringified but get the original error in this case.